pajbot/modules/vip_refresh.py

```diff
@@ -26,6 +27,7 @@ class VIPRefreshModule(BaseModule):
     def __init__(self, bot):
         super().__init__(bot)
         self.scheduled_job = None
+        self.username_regex = re.compile(r"^[a-z0-9](?:\w{1,24})?$", re.IGNORECASE)
```

Comment:
I would prefer if this was static and not stored on the instance

Reply:
Oh, it should also be called LOGIN_REGEX or something like that, since I want to be strict about separating `id`, `login` and `name`

Reply:
See for example https://github.com/pajbot/pajbot/blob/3610d47181be2b3d376c286e8b91bc52d75574f3/pajbot/models/user.py#L43

Reply:
Seems good :+1: